requirements.txt

```diff
@@ -1,3 +1,3 @@
-elasticsearch==6.3.0
-pysolr==3.3.3
+elasticsearch>=6.0.0,<7.0.0
+pysolr>=3.3.3,<4.0
```

Comment:
No, the intention was to at least be on 3.3.3 because that's where cursor support was added. I tacked on < 4.0 assuming breaking changes wouldn't be introduced in 3.0. Maybe overly optimistic?